zerver/lib/user_groups.py

```diff
@@ -240,8 +240,10 @@ def update_or_create_user_group_for_setting(
     direct_members: List[int],
     direct_subgroups: List[int],
     current_setting_value: Optional[UserGroup],
-) -> UserGroup:  # nocoverage
-    if current_setting_value is not None and not hasattr(current_setting_value, "named_user_group"):
+) -> UserGroup:
+    if current_setting_value is not None and not hasattr(
+        current_setting_value, "named_user_group"
+    ):  # nocoverage
```

Comment:
This change looks like a linter fix from a previous commit?

Reply:
No. The formatting changed due to adding "nocoverage'.

Reply:
Ahh, OK, makes sense.